Depentency bug

npx deprecation-crawler

npx: installed 83 in 10.953s
Cannot find module '@nrwl/workspace'
Require stack:
- C:\Users\micha\AppData\Roaming\npm-cache\_npx\15968\node_modules\deprecation-crawler\lib\output-formatters\generate-markdown.js
- C:\Users\micha\AppData\Roaming\npm-cache\_npx\15968\node_modules\deprecation-crawler\lib\output-formatters\index.js
- C:\Users\micha\AppData\Roaming\npm-cache\_npx\15968\node_modules\deprecation-crawler\lib\deprecation-crawler.js
- C:\Users\micha\AppData\Roaming\npm-cache\_npx\15968\node_modules\deprecation-crawler\index.js
- C:\Users\micha\AppData\Roaming\npm-cache\_npx\15968\node_modules\deprecation-crawler\cli.js

--- a/packages/deprecation-crawler/src/lib/output-formatters/generate-markdown.ts
+++ b/packages/deprecation-crawler/src/lib/output-formatters/generate-markdown.ts
@@ -1,10 +1,9 @@
 import { writeFileSync } from 'fs';
 import { basename, join } from 'path';
 import { CrawlConfig, Deprecation } from '../models';
-import { ensureDirExists } from '../utils';
+import { ensureDirExists, toFileName } from '../utils';
 import { EOL } from 'os';
-import { checkout } from "../checkout";
-import { toFileName } from "@nrwl/workspace";
+import { checkout } from '../checkout';
 
 export async function generateTagBasedFormatter(
   config: CrawlConfig,
@@ -70,10 +69,7 @@ export async function generateTagBasedFormatter(
 
   ensureDirExists(config.outputDirectory);
   writeFileSync(
-    join(config.outputDirectory, `${toFileName(config.gitTag
-      .split('/').join('')
-      .split('\\').join('')
-    )}.md`),
+    join(config.outputDirectory, `${toFileName(config.gitTag)}.md`),
     markdownContent
   );
   console.log('Updated tag-based markdown format');

--- a/packages/deprecation-crawler/src/lib/processors/grouping.ts
+++ b/packages/deprecation-crawler/src/lib/processors/grouping.ts
@@ -1,9 +1,13 @@
 import { EOL } from 'os';
 import { prompt } from 'enquirer';
-import { CrawlConfig, CrawledRelease, CrawlerProcess, Deprecation } from "../models";
-import { concat, tap, updateRepoConfig } from "../utils";
-import { toFileName } from "@nrwl/workspace";
-import { generateRawJson } from "../output-formatters";
+import {
+  CrawlConfig,
+  CrawledRelease,
+  CrawlerProcess,
+  Deprecation,
+} from '../models';
+import { concat, tap, updateRepoConfig, toFileName } from '../utils';
+import { generateRawJson } from '../output-formatters';
 
 const ungrouped = 'ungrouped';
 
@@ -12,16 +16,18 @@ interface Group {
   matchers: string[];
 }
 
-export function group(config: CrawlConfig): CrawlerProcess<CrawledRelease, CrawledRelease> {
+export function group(
+  config: CrawlConfig
+): CrawlerProcess<CrawledRelease, CrawledRelease> {
   return concat([
     async (r: CrawledRelease) => ({
       ...r,
-      deprecations: await addGrouping(config, r.deprecations)
+      deprecations: await addGrouping(config, r.deprecations),
     }),
     tap((r: CrawledRelease) =>
       generateRawJson(config, r.deprecations, { tagDate: r.date })
-    )
-  ])
+    ),
+  ]);
 }
 
 export async function addGrouping(
@@ -119,9 +125,9 @@ async function getGroupNameFromExistingOrInputQuestion(
   const isExistingGroup = answerNameFromExisting.existingKey !== newGroupChoice;
   return isExistingGroup
     ? answerNameFromExisting.existingKey
-    : await prompt([getGroupNameQuestion(deprecation)]).then(
-        (s: { key: string }) => toFileName(s.key)
-      );
+    : await prompt([
+        getGroupNameQuestion(deprecation),
+      ]).then((s: { key: string }) => toFileName(s.key));
 }
 
 function getGroupNameFromExistingQuestion(
@@ -184,7 +190,8 @@ function testMessage(reg: string, deprecationMessage: string): boolean {
 // Used for both, message and regex string.
 // Otherwise it could happen that will never match
 function parseDeprecationMessageOrRegex(deprecationMessage: string): string {
-  return deprecationMessage
+  return (
+    deprecationMessage
       // check against lowercase to avoid multiple patterns
       .toLowerCase()
       // normalize multiple whitespaces to one
@@ -192,5 +199,6 @@ function parseDeprecationMessageOrRegex(deprecationMessage: string): string {
       .filter((s) => !!s)
       .join(' ')
       // tri trailing and leading white spaces
-      .trim();
-};
+      .trim()
+  );
+}

--- a/packages/deprecation-crawler/src/lib/utils.ts
+++ b/packages/deprecation-crawler/src/lib/utils.ts
@@ -39,11 +39,24 @@ export function updateRepoConfig(config: CrawlConfig) {
   writeFileSync(path, prettiedConfig);
 }
 
+/**
+ * Ensures the file exists before reading it
+ */
 export function readFile(path: string) {
   if (existsSync(path)) {
     return readFileSync(path, 'utf-8');
   }
   return '';
+}
+
+/**
+ * Upper camelCase to lowercase, hypenated
+ */
+export function toFileName(s: string): string {
+  return s
+    .replace(/([a-z\d])([A-Z])/g, '$1_$2')
+    .toLowerCase()
+    .replace(/[ _]/g, '-');
 }
 
 export function concat<I>(
@@ -99,7 +112,9 @@ export async function getCurrentHeadName(): Promise<string> {
   // That will output the value of HEAD,
   // if it's not detached, or emit the tag name,
   // if it's an exact match. It'll show you an error otherwise.
-  return git(['symbolic-ref -q --short HEAD || git describe --tags --exact-match']);
+  return git([
+    'symbolic-ref -q --short HEAD || git describe --tags --exact-match',
+  ]);
 }
 
 export function cmd(command: string, args: string[]): Promise<string> {

